src/components/BugBountyPoints.tsx

```diff
@@ -75,7 +75,7 @@ const BugBountyPoints: React.FC<IProps> = () => {
           hasError: true,
         })
       })
-  }, [])
+  }, [state])
```

Comment:
@pettinarip ah right! Better I think would be to update `setState` to use its updater callback for previous state. Then there would be no worry about `state` as a dep because it will not be directly used. Same goes for the other components in question.

Reply:
True. That would be better @TylerAPfledderer